spec/system/kudos/kudo_manage_spec.rb

```diff
@@ -14,6 +14,7 @@
     click_button 'Edit'
     fill_in 'Title', with: 'A new title'
     fill_in 'Content', with: 'A new content'
+    page.select kudo.company_value.title, from: 'kudo_company_value_id'
```

Comment:
it would be nice to create at the beginning of the test new company_value, and change here from old to new, and check at the end of the test if new company_value was saved correctly and is visible on index or show page